devstack/settings

```diff
@@ -37,11 +37,23 @@ enable_service group-policy
 disable_service tempest
 ENABLED_SERVICES+=,heat,h-api,h-api-cfn,h-api-cw,h-eng
 # NFP services
-enable_service orchestrator,proxy,proxy_agent,base_configurator
+enable_service orchestrator,proxy,proxy_agent,config_orch
```

Comment:
But we have removed all the base mode functions from lib/nfp and plugin.sh.
So, wouldn't this distinction be meaningless?

Regards,
Deepak

On Fri, May 6, 2016 at 1:54 PM, Rajendra Machani <notifications@github.com>
wrote:

> In devstack/settings
> <https://github.com/oneconvergence/group-based-policy/pull/283#discussion_r62301223>
> :
>
> > @@ -37,11 +37,23 @@ enable_service group-policy
> >  disable_service tempest
> >  ENABLED_SERVICES+=,heat,h-api,h-api-cfn,h-api-cw,h-eng
> >  # NFP services
> > -enable_service orchestrator,proxy,proxy_agent,base_configurator
> > +enable_service orchestrator,proxy,proxy_agent,config_orch
>
> How can we distinguish between the mode of installation, base mode or
> advanced mode ?
> based on that we need to enable specific services.
> for base mode, base_configurator
> for advanced mode, config_orch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/283/files/388c79ce352f50d3ce063e38269f9045a835e634#r62301223>
>